Backend/app/dashboard/provider-dashboard/src/Domain/Action/ProviderPlatform/Management/DriverRegistration.hs

```diff
@@ -68,7 +68,7 @@ postDriverRegistrationDocumentUpload merchantShortId opCity apiTokenInfo driverI
   checkedMerchantId <- merchantCityAccessCheck merchantShortId apiTokenInfo.merchant.shortId opCity apiTokenInfo.city
   transaction <- buildTransaction apiTokenInfo (Just driverId) (Just req)
   T.withResponseTransactionStoring transaction $
-    Client.callManagementAPI checkedMerchantId opCity (.driverRegistrationDSL.postDriverRegistrationDocumentUpload) driverId req
+    Client.callManagementAPI checkedMerchantId opCity (.driverRegistrationDSL.postDriverRegistrationDocumentUpload) driverId req {Common.requestorId = Just apiTokenInfo.personId.getId}
```

Comment:
should we use merchant based func?

(_, requestorId) <- getMbFleetOwnerAndRequestorIdMerchantBased apiTokenInfo Nothing

because for existing flow, we should check `fleet_member_association` table:

```
getFleetOwnerId :: Text -> Maybe Text -> Flow Text
getFleetOwnerId memberPersonId mbFleetOwnerId = do
  maybe
    ( FMA.findAllActiveByfleetMemberId memberPersonId True
        >>= \case
          [] -> return memberPersonId
          [DFMA.FleetMemberAssociation {..}] -> return fleetOwnerId
          _ -> throwError AccessDenied
    )
    identity
    ((verifyFleetOwnerAccess memberPersonId) <$> mbFleetOwnerId)
```